build-system/compile/single-pass.js

```diff
@@ -308,10 +306,17 @@ exports.getGraph = function(entryModules, config) {
   });
 
   config.babel = config.babel || {};
+  const babelPlugins = conf
```

Comment:
Can we prioritize https://github.com/ampproject/amphtml/pull/27161? We're adding more and more call-site configurations that will make it difficult to synchronize changes across environments.

Reply:
I believe @rcebulko is already working on this. (A fair bit has changed in the recent two days, so that PR will need a full rebase before proceeding. Shouldn't be bad though.)

Reply:
Can/will do :+1: